src/njoy21/input/DTFR/Card8/Hisnam.hpp

```diff
@@ -0,0 +1,19 @@
+struct Hisnam {
+  using Value_t = optional<std::string>;
```

Comment:
because the user should be able to put in a bunch of card8s if they want, and an empty card will terminate DTFR. Since the other three things that come after hisnam in card8 are optional, and out of fear that an empty string or a space, " ", would be treated as an acceptable entry for hisnam, I wanted to ensure that an empty card, " / ", would not be treated as a valid card8. 

So if we don't get a hisnam entry, or if it's just a space, then we default to a value of nullopt. That way, we have a clear indication whether or not it's a card8 entry, or a blank card.

In DTFR.hpp, you can see around line 70 that as long as we don't get nullopt in for a hisnam entry, then we can continue to pick up card8s.